documentation/modules/managing/con-consumer-config-properties.adoc

```diff
@@ -0,0 +1,252 @@
+// This module is included in the following files:
+//
+// assembly-client-config.adoc
+
+[id='con-consumer-config-properties-{context}']
+= Kafka consumer configuration tuning
+
+Use a basic consumer configuration with optional properties that are tailored to specific use cases.
+
+When tuning your consumers your primary concern will be ensuring that they cope efficiently with the amount of data ingested.
+As with the producer tuning, be prepared to make incremental changes until the consumers operate as expected.
+
+== Basic consumer configuration
+
+Connection and deserializer properties are required for every consumer.
+Generally, it is good practice to add a client id for tracking.
+
+In a consumer configuration, irrespective of any subsequent configuration:
+
+* The consumer fetches from a given offset and consumes the messages in order, unless the offset is changed to skip or re-read messages.
+* The broker does not know if the consumer processed the responses, even when committing offsets to Kafka because the offsets might be sent to a different broker in the cluster.
```

Comment:
```suggestion
* The broker does not know if the consumer processed the responses, even when committing offsets to Kafka, because the offsets might be sent to a different broker in the cluster.
```